### ISSUE TITLE
Question regarding Shared Checkpoint 

### ISSUE THREAD

**Issue by AnushaManila** [syinari0123/tridepth#4]
After training the model, the checkpoints saved in the log folder with '.pth.tar' extension have a size of 754 MB while the shared checkpoint with '.pth' extension has size of 251.5 MB only. Also the inference from the two checkpoints seem to be different. The shared checkpoint generates nicer output. The retrained model generates images with sharper edges of triangles. Please let me know if there are any post processing missing out or any other steps to be executed. Thank you!

**Comment by syinari0123** [comment]
Sorry it took so long to get back to you.
You mean that the model you trained is larger and less accurate than our distributed pretrained model (from https://www.dropbox.com/s/oos76qomuvddu21/weight_upconv.pth). Is it right?

Regarding the size of model, I think that the "pth.tar" file saved in our training code contains not only the weight but also other information (optimizer, etc).
About the accuracy, I wonder if you set the model as evaluation model properly when you inference it. I often forget it :(

**Comment by AnushaManila** [comment]
Thank you for your comment!
Good to know that the shared model contains only necessary info and not the auxiliary data. 
Since I trained the model from scratch, the model  has stored all the data and I didn't train the model for 30 epochs which is probably the reason for different accuracy. 
Hence closing the issue now
